docs/decisions/index.md

```diff
@@ -0,0 +1,15 @@
+# Architectural Decision Log
+
+This log lists the architectural decisions for [project name].
+
+<!-- adrlog -- Regenerate the content by using "adr-log -i". You can install it via "npm install -g adr-log" -->
+
+* [ADR-0000](0000-use-markdown-architectural-decision-records.md) - Use Markdown Architectural Decision Records
+* [ADR-0001](0001-automatically-create-test-configurations-from-source-directory-names.md) - Automatically derive test configurations from source directory names
+* [ADR-0002](0002-automatically-create-shared-test-set.md) - Automatically create 'sharedTest' source set
+
+<!-- adrlogstop -->
+
+For new ADRs, please use [template.md](template.md) as basis.

```

Comment:
should this reference `madr-template.md`?

Reply:
good catch, fixed